metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/usage.py

```diff
@@ -184,6 +186,7 @@ def __init__(self, config: BigQueryV2Config):
                     e.timestamp, config.bucket_duration
                 ),
                 "user": lambda e: e.actor_email,
+                "from_query": lambda e: 1 if e.from_query else 0,
```

Comment:
This can be `lambda e: int(e.from_query)` to match `is_read` on query events. Same goes for `on_view`

Reply:
thanks.